src/main/resources/messages/CxBundle.properties

```diff
@@ -80,7 +80,7 @@ SCAN_FINISHED=Checkmarx scan completed successfully
 SCAN_FINISHED_LOAD_RESULTS=Would you like to load the results?
 LOAD_RESULTS=Loading results for scan id {0}...
 PROJECT_DOES_NOT_MATCH_TITLE=Wrong project
-PROJECT_DOES_NOT_MATCH_QUESTION=The files open in your workspace don't match the files previously scanned in this Checkmarx project. Do you want to scan anyway?
+PROJECT_DOES_NOT_MATCH_QUESTION=Git project doesn't match the selected Checkmarx project. Do you want to scan anyway?
```

Comment:
validate that it same as we have in vscode

Reply:
its samw